service/router.go

```diff
@@ -47,23 +48,41 @@ func SigningRouter() *mux.Router {
 	// Start the web service router
 	router := mux.NewRouter()
 
-	// API routes
 	router.Handle("/v1/version", Middleware(http.HandlerFunc(core.Version))).Methods("GET")
 	router.Handle("/v1/health", Middleware(http.HandlerFunc(core.Health))).Methods("GET")
-	router.Handle("/v1/serial", Middleware(ErrorHandler(sign.Serial))).Methods("POST")
-	router.Handle("/v1/request-id", Middleware(ErrorHandler(sign.RequestID))).Methods("POST")
-	router.Handle("/v1/model", Middleware(ErrorHandler(assertion.ModelAssertion))).Methods("POST")
-	router.Handle("/v1/pivot", Middleware(ErrorHandler(pivot.Model))).Methods("POST")
-	router.Handle("/v1/pivotmodel", Middleware(ErrorHandler(pivot.ModelAssertion))).Methods("POST")
-	router.Handle("/v1/pivotserial", Middleware(ErrorHandler(pivot.SerialAssertion))).Methods("POST")
-	router.Handle("/v1/pivotuser", Middleware(ErrorHandler(pivot.SystemUserAssertion))).Methods("POST")
+
```

Comment:
the only change in this file was to add `metric.CollectAPIStats("callerName" ...` to every route we have here. And some reformating to not have the line too long